engine/api/ascode/sync/sync.go

```diff
@@ -0,0 +1,98 @@
+package sync
+
+import (
+	"context"
+
+	"github.com/go-gorp/gorp"
+
+	"github.com/ovh/cds/engine/api/ascode"
+	"github.com/ovh/cds/engine/api/cache"
+	"github.com/ovh/cds/engine/api/event"
+	"github.com/ovh/cds/engine/api/repositoriesmanager"
+	"github.com/ovh/cds/engine/api/workflow"
+	"github.com/ovh/cds/sdk"
+	"github.com/ovh/cds/sdk/log"
+)
+
+// SyncAsCodeEvent checks if workflow as to become ascode
+func SyncAsCodeEvent(ctx context.Context, db *gorp.DbMap, store cache.Store, proj *sdk.Project, app *sdk.Application, u sdk.Identifiable) ([]sdk.AsCodeEvent, string, error) {
```

Comment:
Unit tests

Reply:
Tested by Test_postResyncPRAsCodeHandler .  ascode_test.go:  db check at the end